Products/ZenModel/migrate/addHeartbeatTimeout.py

```diff
@@ -0,0 +1,81 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2016, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+__doc__ = """
```

Comment:
Technically, `__doc__ = ` is redundant.  Python automatically uses the first global string in a module as the docstring for the module, so it's unnecessary and can be removed.